### ISSUE TITLE
Retention?

### ISSUE THREAD

**Issue by cadilhac** [JudahGabriel/RavenDB.StructuredLog#8]
Is there a way to specify how long logs should stay in the database? Thanks.

**Comment by JudahGabriel** [comment]
Yes. Have a look at [LogOptions.cs](https://github.com/JudahGabriel/RavenDB.StructuredLog/blob/master/RavenDB.StructuredLog/LogOptions.cs)
